src/nl/sensorlab/videowall/LedWallApplication.java

```diff
@@ -21,8 +21,9 @@
 	private WallDriver driver;
 	private Preview preview;
 	private MainWindow mainWindow;
-
-	private boolean ledPreviewEnabled = true;
+	private Rectangle previewRect;
```

Comment:
This Rectangle is the same as the PApplet canvas, so this is a bit superfluous (you can just call PApplet.width() and PApplet.height() to get the sizes.